backend/wine_map/serializers.py

```diff
@@ -0,0 +1,44 @@
+from rest_framework import serializers
+from .models import Wine, Brand, Country, Sweetness, Taste, DishCategory
```

Comment:
For the same reason as above

```suggestion

from .models import Wine, Brand, Country, Sweetness, Taste, DishCategory
```